components/observations/ObservationsListView.tsx

```diff
@@ -71,9 +74,28 @@ export const ObservationsListView: React.FunctionComponent<ObservationsListViewP
   const originalFilterConfig: ObservationFilterConfig = useMemo(() => createDefaultFilterConfig(additionalFilters), [additionalFilters]);
   const [filterConfig, setFilterConfig] = useState<ObservationFilterConfig>(originalFilterConfig);
   const [filterModalVisible, {set: setFilterModalVisible, on: showFilterModal}] = useToggle(false);
+  const {data: metaData} = useAvalancheCenterMetadata(center_id);
```

Comment:
Please keep the result pattern and add this to the `incompleteQueryState` block.